### ISSUE TITLE
Question to Cindy about GH Pages video

### ISSUE THREAD

**Issue by mblack88** [mblack88/Conversations#2]
@crichID

I saw a "GitHub" pages video last week, that I'm pretty sure that you did.
In it you did `git checkout --detach gh-pages`.  I am wondering why you did the `detach`, what benefit would it have?

TIA

**Comment by crichID** [comment]
Hey @mblack88 :wave: 

In that video we did a `git checkout --orphan gh-pages` which essentially creates a new branch with a unique history. Instead of sharing refs with master, the gh-pages branch created with this method will have it's own history. This is most commonly used when you want to have your main project on master and then keep your project documentation on a secondary branch (and served on GitHub Pages). To be honest, most projects create a second repo for the documentation, but it is a second workflow option :wink:

Let me know if this still doesn't make sense and I can try again!

**Comment by mblack88** [comment]
Makes sense, I sort of thought that. Hope training today went well, and my old team didn't give you to much trouble.

Sent from my iPhone

On Sep 27, 2016, at 5:14 PM, Cynthia Rich <notifications@github.com<mailto:notifications@github.com>> wrote:

Hey @mblack88https://github.com/mblack88 ?

In that video we did a git checkout --orphan gh-pages which essentially creates a new branch with a unique history. Instead of sharing refs with master, the gh-pages branch created with this method will have it's own history. This is most commonly used when you want to have your main project on master and then keep your project documentation on a secondary branch (and served on GitHub Pages). To be honest, most projects create a second repo for the documentation, but it is a second workflow option ?

Let me know if this still doesn't make sense and I can try again!

##
